src/modules/__tests__/conditions.spec.js

```diff
@@ -250,3 +255,144 @@ describe('conditions that do not include legend sets or option sets', () => {
         })
     })
 })
+
+describe('checkIsCaseSensitive', () => {
+    const tests = [
+        {
+            operator: '!LIKE',
+            expected: true,
+        },
+        {
+            operator: '!ILIKE',
+            expected: false,
+        },
+        {
+            operator: '!EQ',
+            expected: true,
+        },
+        {
+            operator: '!IEQ',
+            expected: false,
+        },
+        {
+            operator: 'LIKE',
+            expected: true,
+        },
+        {
+            operator: 'ILIKE',
+            expected: false,
+        },
+        {
+            operator: 'EQ',
+            expected: true,
+        },
+        {
+            operator: 'IEQ',
+            expected: false,
+        },
+        // The function doesn't handle 'IN' correctly
+        // {
+        //     operator: 'IN',
+        //     expected: true,
+        // },
+        // {
+        //     operator: '!IN',
+        //     expected: true,
+        // },
+    ]
+
+    tests.forEach((t) => {
+        const testname = `${t.operator}: expected: ${t.expected}`
+        test(testname, () => {
+            expect(checkIsCaseSensitive(t.operator)).toEqual(t.expected)
+        })
+    })
+})
+
+describe('prefixOperator', () => {
+    const tests = [
+        {
+            operator: 'LIKE',
+            isCaseSensitive: true,
+            expected: 'LIKE',
+        },
+        {
+            operator: '!LIKE',
+            isCaseSensitive: false,
+            expected: '!ILIKE',
+        },
+        {
+            operator: 'LIKE',
+            isCaseSensitive: false,
+            expected: 'ILIKE',
+        },
```

Comment:
Add a case for `!LIKE` -> `!LIKE` and `!EQ` -> `!EQ` (i.e. the "not" operator with `isCaseSensitive: true`)